extensions/parquet/table/src/main/java/io/deephaven/parquet/table/layout/ParquetSingleFileLayout.java

```diff
@@ -34,6 +38,7 @@ public String toString() {
 
     @Override
     public void findKeys(@NotNull final Consumer<ParquetTableLocationKey> locationKeyObserver) {
-        locationKeyObserver.accept(new ParquetTableLocationKey(parquetFileUri, 0, null, readInstructions));
+        locationKeyObserver
```

Comment:
We should write down the key at construction time. And maybe delete this class in favor of supplying one key to a `KnownKeys` provider.